### ISSUE TITLE
tag cloud

### ISSUE THREAD

**Issue by norayr** [diaspora/diaspora#6187]
diaspora should generate tag cloud in user's profile.
also, by clicking on the tag user should get list of posts by this user tagged by that tag, not all the posts of all users tagged by that tag.

**Comment by norayr** [comment]
this is of course suggestion.

**Comment by atulonruby** [comment]
do you have any desgin for tag cloud in mind

**Comment by norayr** [comment]
no, I am not a ux person. I believe any tag cloud, like wordpress has, will suit me.

 I just need navigation on user's posts via tags. Otherwise If I click on tag, I get all the posts of all users marked with that tag. If I want to see what particular user have been written/shared about this it becomes really hard.

**Comment by cmrd-senya** [comment]
That would require a feature to search a post with tags for a specific user. VK.com uses a specific syntax for that feature. They have #tag@page_name - that means show only posts with the tag "tag" for the user "page_name". We need something similar. #tag@id@pod.domain could be used, but double @ looks a little clumsy and can be mixed up with mentions, but I think it's not a big deal.

**Comment by norayr** [comment]
it would be good to have specific syntax to search for user's tag in the search box, but having a tag cloud at the user's profile pane would not require special syntax knowledge and will allow easy navigation.

**Comment by cmrd-senya** [comment]
I think it just the way it should be implemented. Like, if we use
```#tag@user@pod.d``` as the syntax, then the URL
```https://pod/tags/tag@user@pod.d``` will show a search for tags of
specific users, and the sound cloud can link to URLs of a such look.

**Comment by atulonruby** [comment]
<img width="1374" alt="screen shot 2015-12-01 at 11 07 29 pm" src="https://cloud.githubusercontent.com/assets/1123985/11508849/dd0bc376-9880-11e5-8c4e-9d7fbd6b3164.png">
Will it be a good idea to make something like this.

**Comment by jhass** [comment]
Not sure. How about as a tab at the top and then you get a popover with it?

**Comment by jaywink** [comment]
@atulonruby nice! I kinda agree it should be in the "content", maybe have the tag cloud scroll in view under the publisher by clicking a "tags" link etc. The left bar is for profile content. But would be so great to have this!

**Comment by atulonruby** [comment]
@jaywink  I didnt exaclty get where you want the tags to be. Do we want it as a list from a  dropdown.
And in that case, where we want the dropdown to be.

**Comment by jaywink** [comment]
@atulonruby personally I love the one you did above - but as @jhass also said, maybe put it inside a tab (between Posts and Contacts) or make it slide down on click (maybe a button/link) between the publisher and stream). Having it where the profile information is currently is not good imho, as otherwise it will either hide profile info or get hidden by it.

No dropdowns :D The cloud concept looks good, just placement.

**Comment by norayr** [comment]
I would like to add here what was my motive to open this.

I believe we have two major problems in Diaspora, one of them is not supporting decentralization technologies (for instance openid is neither provided nor people allowed to comment with it), and the other problem is navigation.

Diaspora lacks good navigation (in my opinion of course). It is very hard to find your own post about something.

in other places help tags (not like in diaspora), calendar, or pages.

tags are may be not the best way to find your own post, but at least they exist 
 If you know the tag, by the tag you get a lot of posts, tagged by this tag, not necessarily your posts.

and you cannot find your own post in the sea of other posts.

also other users cannot search for your posts tagged with particular tag.

that's why i think that the tag cloud on the user's page is a good idea. because that would allow to search within user's tags. and it's already well known to users.

**Comment by norayr** [comment]
by the way tag cloud does not solve navigation problem in case author decided to not use tags in her post.

**Comment by versus167** [comment]
uhh - 3 years old.. But interesting thing
